build.gradle

```diff
@@ -192,10 +195,20 @@ dependencyUpdates.resolutionStrategy {
 
 // https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
 dependencyCheck {
-  // Specifies if the build should be failed if a CVSS score above a specified level is identified.
-  // range of 7-10 fails the build, anything lower and it doesn't fail the build
-  failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'true' ? 7.0 : 11
+  // If we set a dedicated environment variable to 'false' to be lax in our build standards, then we won't fail the build,
+  //  otherwise we are strict in our build and fail it.
+  failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'false' ? 11 : 7.0
+  cveValidForHours = 4 // on Jenkins we have 24, so if you get more vulnerabilities locally, try to normalize this value with Jenkins
   suppressionFile = 'config/owasp/suppressions.xml'
+
+  analyzers {
+    retirejs {
+      enabled = false
+    }
+    ossIndex {
+      enabled = false
+    }
+  }
```

Comment:
This section makes the behaviour observed when running gradle's dependencyCheck tasks locally identical to that of the deployed versions. Recommend bringing this change in other projects too.